packages/spec-parser/package.json

```diff
@@ -49,9 +49,9 @@
     "access": "public"
   },
   "devDependencies": {
+    "@apidevtools/json-schema-ref-parser": "^9.0.6",
```

Comment:
this dev dependencies is needed?

Reply:
This line is removed in previous pr, restore it for keep minimum changes from last stable version and avoid unexpected error.